eng/pipelines/mono/templates/build-job.yml

```diff
@@ -81,6 +74,12 @@ jobs:
         value: wasm
       - name: osOverride
         value: '-os Browser'
+    - ${{ if eq(parameters.runtimeVariant, 'llvmjit') }}:
+      - name: llvmParameter
+        value: /p:MonoEnableLLVM=true
```

Comment:
One comment: if I change Mono to search for opt/llc at a specific relative path by default, then MonoBundleLLVMOptimizer should probably be "true" by default (and explicitly disabled for no-llvm-aot builds), so that the default LLVM developer workflow doesn't require manually setting another parameter.

Reply:
True by default for MonoEnableLLVM, maybe, but definitely not true by default in the general case?

Reply:
Definitely not true by default in the general case.